tests/s25Main/campaign/testCampaignLuaFile.cpp

```diff
@@ -89,7 +90,7 @@ BOOST_AUTO_TEST_CASE(LoadCampaignDescriptionWithoutTranslation)
         bnw::ofstream file(tmp.get() / "campaign.lua");
 
         file << "campaign ={\
-            version = \"1\",\
+            version = \"2\",\
```

Comment:
Similar: Why? Where is this used and why do existing campaigns need to be upgraded if they don't use anything new and the new feature is optional?

Reply:
Yes i mixed up the lua version and the campaign version